Documentation/DSCResources/VMHostFirewallRuleset/VMHostFirewallRuleset.md

```diff
@@ -0,0 +1,64 @@
+# VMHostFirewallRuleset
+
+## Parameters
+
+| Parameter | Attribute | DataType | Description | Allowed Values |
+| --- | --- | --- | --- | --- |
+| **Server** | Key | string | The name of the Server we are trying to connect to. The Server can be a vCenter or ESXi. ||
+| **Credential** | Mandatory | PSCredential | The credentials needed for connection to the specified Server. ||
+| **VMHostName** | Key | string | The name of the VMHost. ||
+| **Name** | Key | string | The name of the firewall ruleset. ||
+| **Enabled** | Optional | bool | Specifies whether the firewall ruleset should be enabled or disabled. ||
+| **AllIP** | Optional | bool | Specifies whether the firewall ruleset allows connections from any IP address. ||
```

Comment:
Isn;t better to be named AnyIP? the description states from any IP, why the name is ALL? It leaves the impression the property is regarding the ESX  IP not the connection source IP

Reply:
Well **AllIP**, because connections are allowed from any IP address, so every IP will be valid. IP addresses can be restricted with the **IPAddresses** property.